NEWS.md

```diff
@@ -1,5 +1,14 @@
 # goshawk 0.1.13.9009
 
+### Fixes
+* Fixed `h_caption_loqs_label` to handle multiple parameters when data frame is returned from `h_identify_loq_values`.
+
+### Enhancements
+* Implemented `nestcolor` in goshawk.
```

Comment:
Actually, there is a NEWS entry for `nestcolor` that was wrongly entered in the past release: https://github.com/insightsengineering/goshawk/pull/165/files

Please remove the wrong entry or replace the above with it.

Reply:
This is not the first time I am seeing nestcolor related news entry inputted in the past release. May I ask you to double-check that for other packages as well?

Reply:
Thanks for catching this.
I'll replace the above with it.